pom.xml

```diff
@@ -0,0 +1,608 @@
+                <argument>-jar</argument>
+                <argument>
+                  ${project.build.directory}/${project.build.finalName}-SoyMsgExtractor.jar
+                </argument>
+                <argument>--outputFile</argument>
+                <argument>${soy.examples.out}/examples_extracted.xlf
+                </argument>
+                <argument>--inputPrefix</argument>
+                <argument>${soy.examples}/</argument>
+                <argument>simple.soy</argument>
+                <argument>features.soy</argument>
+              </arguments>
+            </configuration>
+            <goals>
+              <goal>exec</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <version>1.3.1</version>
+        <executions>
+          <execution>
+            <id>enforce-files-exist</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <phase>verify</phase>
+            <configuration>
+              <rules>
+                <requireFilesExist>
+                  <files>
+                    <file>${soy.examples.out}/simple_generated.js</file>
+                    <file>${soy.examples.out}/simple_generated_en.js</file>
+                    <file>${soy.examples.out}/features_generated_en.js</file>
+                    <file>${soy.examples.out}/simple_generated_x-zz.js</file>
+                    <file>${soy.examples.out}/features_generated_x-zz.js</file>
+                    <file>${soy.examples.out}/FeaturesSoyInfo.java</file>
+                    <file>${soy.examples.out}/examples_extracted.xlf</file>
+                  </files>
+                </requireFilesExist>
+              </rules>
+              <fail>true</fail>
+            </configuration>
+          </execution>
+          <execution>
+            <id>enforce</id>
+            <configuration>
+              <rules>
+                <DependencyConvergence/>
+              </rules>
+              <fail>true</fail>
+            </configuration>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-source-plugin</artifactId>
+        <version>2.2.1</version>
+        <executions>
+          <execution>
+            <id>attach-sources</id>
+            <goals>
+              <goal>jar</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-javadoc-plugin</artifactId>
+        <version>2.9.1</version>
+        <executions>
+          <execution>
+            <id>attach-javadocs</id>
+            <goals>
+              <goal>jar</goal>
```

Comment:
nit: isn't the execution configured already by the `oss-parent` parent? (i.e. we'd only need the `<configuration>` at the `<plugin>` level rather than `<execution>` level)